sample-graphs/update.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -eu
+
+SOURCECRED_REMOTE="git@github.com:sourcecred/sourcecred.git"
+
+# Taken from https://stackoverflow.com/a/246128
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
Whoa, is this intended to be sourced? Prefer to avoid `BASH_SOURCE`
unless there’s a reason: (a) it’s bash-specific, not POSIX, and
(b) you’ll confuse readers (like me) into thinking that there _is_ a
reason.

Happily, you can do away with this entirely (see below).

Reply:
Done.